x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/population_field/description.tsx

```diff
@@ -20,13 +20,11 @@ export const Description: FC = memo(({ children }) => {
       description={
         <FormattedMessage
           id="xpack.ml.newJob.wizard.pickFieldsStep.populationField.description"
-          defaultMessage="All values in the selected field will be modeled together as a population. This analysis type is recommended for high cardinality data."
+          defaultMessage="All values in the selected field will be modeled together as a population. Recommended for high cardinality data."
```

Comment:
Is the `Recommended for high cardinality data.` adding any value here? The user has already decided to create a population job at this point.

<img width="700" alt="image" src="https://user-images.githubusercontent.com/7405507/214636100-f72ed150-7e67-4626-a98f-5f4c6dda81cd.png">


Are we saying they need to pick a high cardinality field here? I think we are trying to say that a population job is recommended for high cardinality data, in which case that sentence would be better off in the step where the user decides what sort of job to create. But that would result in quite a long string of text in that card:

<img width="1073" alt="image" src="https://user-images.githubusercontent.com/7405507/214636426-4f1961dc-fb59-4d5a-b8da-e5a196cbc246.png">

Reply:
Thanks for noticing it, that sentence is indeed out of place. I removed it from the population field description and added it to the population job description. To make the description short, I edited the first sentence there as well. It reads: "Detect unusual activity in a population. Recommended for high cardinality data." 

![Screenshot 2023-01-26 at 10 40 35](https://user-images.githubusercontent.com/22324794/214804973-c3918a3b-b2f7-4ab0-ace3-01b39f7cfa4c.png)

I don't think that the sentence "Detect activity that is unusual compared to the behavior of the population." explains the concept much clearer than the shorter sentence, so I would go with the solution this PR offers. Please let me know if you think differently.

([3307583](https://github.com/elastic/kibana/pull/149407/commits/3307583f8964bf9066984a3f6479cc93972f3115))

Reply:
I shortened the influencer field description via [bf340ed](https://github.com/elastic/kibana/pull/149407/commits/bf340ed01f8918bba941352677d10027bdae6781).